### PR TITLE
updated roadmap

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

**Comment by jasonstephen15** [comment]
Already reviewed by pm team & dev manager, ready for docs team review

File: microsoft-edge/webview2/roadmap.md [hunk 1]
```diff
@@ -23,32 +23,16 @@ If you have concerns or questions about the Roadmap, provide your feedback in th
...
+H2 2021:
+* Desktop UWP Preview
+
+H1 2022:
+* Desktop UWP GA
+
+H2 2022 & Beyond:
+* MacOS Preview
+* Xbox Preview
+* HoloLens Preview
```
Comment (Reezaali) [inline microsoft-edge/webview2/roadmap.md:35]: @jasonstephen15  - re: MacOS, Xbox, and HoloLens - are these related to UWP releases?

File: microsoft-edge/webview2/roadmap.md [hunk 2]
```diff
@@ -68,12 +52,18 @@ The Win32 C/C++ SDK has reached GA.
 
 The .NET SDK has reached GA. 
 
-### WinUI 3.0  
+### WinUI 3
+
+You can access WebView2 in your applications using [Win UI 3][UwpToolkitsWinui3Index] via WindowsAppSDK. This is currently in preview with GA set for October. For more info visit the WindowsAppSDK [roadmap][WindowsAppSDKRoadmap].
```
Comment (Reezaali) [inline microsoft-edge/webview2/roadmap.md:57]:
```suggestion
You can access WebView2 controls in your applications using [Win UI 3][UwpToolkitsWinui3Index] with the Windows App SDK. This is currently in preview. For more information, navigate to the [Windows App SDK roadmap][WindowsAppSDKRoadmap].
```

File: microsoft-edge/webview2/roadmap.md [hunk 3]
```diff
@@ -68,12 +52,18 @@ The Win32 C/C++ SDK has reached GA.
...
+
+### WinUI 2
 
-You can access WebView2 in your UWP applications using [Win UI 3.0][UwpToolkitsWinui3Index], currently in alpha.  For more information about keeping up to date, navigate to [Windows UI Library Roadmap][GithubMicrosoftUiXamlRoadmap].  
+WebView2 will be supported in UWP applications via WinUI 2. The team is aiming for a public preview in H2 2021. More info to come!
```
Comment (Reezaali) [inline microsoft-edge/webview2/roadmap.md:61]:
```suggestion
WebView2 will be supported in UWP applications using WinUI 2, which will be in preview later this year.
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/roadmap.md (hunk 1 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/roadmap.md:35]: No, they are seperate releases we don't have too much more specifc info about

**Comment by jasonstephen15** [comment]
We are good to merge @Reezaali

**Review by Reezaali — APPROVED** [review]
